google-cloud-vision/synth.py

```diff
@@ -0,0 +1,38 @@
+import synthtool as s
+import synthtool.gcp as gcp
+import logging
+from pathlib import Path
+import subprocess
+import os
+
+
+logging.basicConfig(level=logging.DEBUG)
+
+gapic = gcp.GAPICGenerator()
+
+v1_library = gapic.ruby_library(
+    'vision', 'v1', artman_output_name='google-cloud-ruby/google-cloud-vision'
+)
+
+s.copy(v1_library / 'lib/google/cloud/vision/v1')
+s.copy(v1_library / 'lib/google/cloud/vision/v1.rb')
+
+s.replace(
+    "test/google/cloud/vision/v1/image_annotator_client_test.rb",
+    "assert_not_nil",
+    "refute_nil")
+
+s.replace(
+    "lib/google/cloud/vision/v1/image_annotator_client.rb",
+    "google/cloud/vision/credentials",
+    "google/cloud/vision/v1/credentials")
+
+s.replace(
```

Comment:
These should be annotated as well. Especially since they're one-time changes, and should be removed immediately to avoid future confusion. Alternately, we could just get rid of them, and make these changes manually in this PR. Might be worth bringing up a question to Thea and Chris about whether we think one-time changes like this belong in synth.py.